react-native/react-native.d.ts

```diff
@@ -1931,6 +1931,14 @@ declare namespace  __React {
     export interface DrawerLayoutAndroidStatic extends React.ComponentClass<DrawerLayoutAndroidProperties> {
 
         /**
+         * Specifies the side of the screen from which the drawer will slide in.
+         */
+        positions: { 
```

Comment:
I'm not really sure what you're suggesting.... 


Traced the values of Left/Right back from: 

[DrawerLayoutAndroid.android.js](https://github.com/facebook/react-native/blob/master/Libraries/Components/DrawerAndroid/DrawerLayoutAndroid.android.js)
var DrawerConsts = UIManager.AndroidDrawerLayout.Constants;

-- [UIManager.js](https://github.com/facebook/react-native/blob/master/Libraries/Utilities/UIManager.js)
const { UIManager } = NativeModules;

---- [NativeModules.js](https://github.com/facebook/react-native/blob/master/Libraries/BatchedBridge/BatchedBridgedModules/NativeModules.js)

From there it looks like it's dynamically loaded from native code:
[ReactDrawerLayoutManager.java](https://github.com/facebook/react-native/blob/master/ReactAndroid/src/main/java/com/facebook/react/views/drawer/ReactDrawerLayoutManager.java)

```
  @Override
  public @Nullable Map getExportedViewConstants() {
    return MapBuilder.of(
        "DrawerPosition",
        MapBuilder.of("Left", Gravity.START, "Right", Gravity.END));
  }
```